vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryController.kt

```diff
@@ -96,9 +98,23 @@ class SpaceDirectoryController @Inject constructor(
                     ?: emptyList()
 
             if (flattenChildInfo.isEmpty()) {
-                genericFooterItem {
-                    id("empty_footer")
-                    stringProvider.getString(R.string.no_result_placeholder)
+                genericEmptyWithActionItem {
+                    id("empty_res")
+                    title(stringProvider.getString(R.string.this_space_has_no_rooms))
+                    iconRes(R.drawable.ic_empty_icon_room)
+                    iconTint(colorProvider.getColorFromAttribute(R.attr.riotx_reaction_background_on))
+                    apply {
```

Comment:
I do not think using a `apply` block is necessary here.

Reply:
done